src/onelogin/saml2/response.py

```diff
@@ -47,7 +47,7 @@ def __init__(self, settings, response):
             self.encrypted = True
             self.decrypted_document = self.__decrypt_assertion(decrypted_document)
 
-    def is_valid(self, request_data, request_id=None):
+    def is_valid(self, request_data, request_id=None, raises=False):
```

Comment:
We should add this functionality to rest of is_valid methods (logoutrequest and logoutresponse)